src/Windows/Avalonia.Win32/OleDataObject.cs

```diff
@@ -44,9 +44,77 @@ public IEnumerable<string> GetFileNames()
 
         public object Get(string dataFormat)
         {
+            if (dataFormat == ClipboardFormats.GetFormat(StandardClipboardFormats.CF_BITMAP))
+            {
+                return GetImage();
+            }
+
             return GetDataFromOleHGLOBAL(dataFormat, DVASPECT.DVASPECT_CONTENT);
         }
 
+        private object GetImage()
+        {
+            // read bitmapInfo
+            var bitmapInfoData = GetDataFromOleHGLOBAL(ClipboardFormats.GetFormat(StandardClipboardFormats.CF_DIB), DVASPECT.DVASPECT_CONTENT) as byte[];
+
+            if (bitmapInfoData == null)
+            {
+                return null;
+            }
+
+            var bitmapInfo = ReadBitmapInfo(bitmapInfoData);
+
+            // get bitmap handle HBITMAP
+            var formatEtc = new FORMATETC();
+            formatEtc.cfFormat = StandardClipboardFormats.CF_BITMAP;
+            formatEtc.dwAspect = DVASPECT.DVASPECT_CONTENT;
+            formatEtc.lindex = -1;
+            formatEtc.tymed = TYMED.TYMED_GDI;
+
+            if (_wrapped.QueryGetData(ref formatEtc) == 0)
+            {
+                _wrapped.GetData(ref formatEtc, out STGMEDIUM medium);
+                try
+                {
+                    if (medium.unionmember != IntPtr.Zero && medium.tymed == TYMED.TYMED_GDI)
+                    {
+                        // get image bytes
+                        var bufer = new byte[bitmapInfo.biSizeImage];
+                        UnmanagedMethods.GetBitmapBits(medium.unionmember, bufer.Length, bufer);
+                        return bufer;
```

Comment:
Are you sure that returning a byte array with raw bitmap data but without pixel format and layout is OK? How does it work in WPF?